PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4083,15 +4086,22 @@ void Finder::addSearchHitCount(int count, int countSearched, bool isMatchLines,
 	setFinderReadOnly(true);
 }
 
-void Finder::add(FoundInfo fi, SearchResultMarking mi, const TCHAR* foundline)
+void Finder::add(FoundInfo fi, SearchResultMarking mi, const TCHAR* foundline, size_t totalLineNumber)
 {
 	_pMainFoundInfos->push_back(fi);
 
 	generic_string str = TEXT("\t");
 	str += _prefixLineStr;
 	str += TEXT(" ");
 
-	str += std::to_wstring(fi._lineNumber);
+	size_t totalLineNumberDigit = size_t(log10(totalLineNumber) + 1);
+	size_t currentLineNumberDigit = size_t(log10(fi._lineNumber) + 1);

```

Comment:
1. Use `static_cast<size_t>(xxxx)` instead of `size_t(xxxx)`.
Use [nbDigitsFromNbLines ](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/MISC/Common/Common.cpp#L1376) instead of `log10`.